docs/src/research/installation.md

```diff
@@ -0,0 +1,71 @@
+<!--
+    Copyright 2022-2023 TII (SSRC) and the Ghaf contributors
```

Comment:
Make sure to update the year here as well.

Reply:
[d593f61](https://github.com/tiiuae/ghaf/commit/d593f614a7fa9ba4fa7471bb55441cccb173c546)